fixtures/etc/fog.json

```
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "fog",
+  "categories": ["Smoke", "Dimmer"],
+  "meta": {
+    "authors": ["oto"],
+    "createDate": "2025-04-02",
+    "lastModifyDate": "2025-04-02"
+  },
+  "links": {
+    "manual": [
+      "https://www.smokemachines.net/data/highend/F100.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [298, 209, 686],
+    "weight": 14,
+    "power": 1450
+  },
+  "availableChannels": {
+    "Fog": {
+      "capability": {
+        "type": "Fog",
+        "fogType": "Fog"
+      }
```

```suggestion
      "capabilities": [
        {
          "dmxRange": [0, 25],
          "type": "Fog",
          "fogType": "Fog",
          "fogOutput": "0%"
        },
        {
          "dmxRange": [26, 255],
          "type": "Fog",
          "fogType": "Fog",
          "fogOutputStart": "0%",
          "fogOutputEnd": "100%"
        }
      ]
```

Manual: "The first 5% of dimmer travel is a zero value."